superset/assets/spec/javascripts/explore/utils_spec.jsx

```diff
@@ -12,48 +13,124 @@ describe('utils', () => {
     expect(uri1.toString()).to.equal(uri2.toString());
   }
 
-  it('getExploreUrl generates proper base url', () => {
+  it('generates proper base url', () => {
```

Comment:
do you think the tests should include the util func name? you do this inconsistently here and less readable if not. you could also create a dedicated `describe` block for a given func if you want to omit the name.

Reply:
fixed. i add `describe` section.